Source/Krypton Components/Krypton.Toolkit/General/PlatformInvoke.cs

```diff
@@ -4556,8 +4556,52 @@ internal struct TITLEBARINFOEX
             public RECT rcHelpButton;
             public RECT rcCloseButton;
         }
+
+        #region For Acrylic
+
+        // The following code comes from https://stackoverflow.com/questions/56481230/how-to-create-windows-10-acrylic-transparency-effect-in-winform-c-sharp
+        // Discovered via:
+        // https://withinrafael.com/2015/07/08/adding-the-aero-glass-blur-to-your-windows-10-apps/
+        // https://github.com/riverar/sample-win32-acrylicblur/blob/917adc277c7258307799327d12262ebd47fd0308/MainWindow.xaml.cs
+
+        [DllImport(Libraries.User32)]
+        public static extern int SetWindowCompositionAttribute(HandleRef hWnd, in WindowCompositionAttributeData data);
+
+        public unsafe struct WindowCompositionAttributeData
+        {
+            public WCA Attribute;
+            public void* Data;
```

Comment:
This should be an `IntPtr` and therefore no unsafe code.
Please **remove** the acrylic work from this PR

Reply:
Commented out ready to go back to later